src/test/components/LocalTrack.test.js

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+
+import {
+  changeSelectedThread,
+  hideLocalTrack,
+} from '../../actions/profile-view';
+import LocalTrack from '../../components/timeline/LocalTrack';
+import {
+  getLocalTracks,
+  getRightClickedTrack,
+} from '../../reducers/profile-view';
+import { getSelectedThreadIndex } from '../../reducers/url-state';
+import { getProfileWithNiceTracks } from '../fixtures/profiles/tracks';
+import { storeWithProfile } from '../fixtures/stores';
+
+const LEFT_CLICK = 0;
+const RIGHT_CLICK = 2;
+
+describe('timeline/LocalTrack', function() {
+  /**
+   *  getProfileWithNiceTracks() looks like: [
+   *    'show [thread GeckoMain process]',
+   *    'show [thread GeckoMain tab]',
+   *    '  - show [thread DOM Worker]',         <- use this local track.
+   *    '  - show [thread Style]',
+   *  ]
+   */
+  function setup() {
+    const pid = 222;
+    const trackIndex = 0;
+    const profile = getProfileWithNiceTracks();
+    const store = storeWithProfile(profile);
+    const { getState, dispatch } = store;
+    const trackReference = { type: 'local', pid, trackIndex };
+    const localTrack = getLocalTracks(getState(), pid)[trackIndex];
+    if (localTrack.type !== 'thread') {
+      throw new Error('Expected a thread track.');
+    }
+    const { threadIndex } = localTrack;
+
+    // The assertions are simpler if this thread is not already selected.
```

Comment:
should we have a test making sure `localTrack.trheadIndex` is selected by default ?

Reply:
I'd say no like with the GlobalTracks.